cfgov/v1/models/base.py

```diff
@@ -354,7 +354,7 @@ def page_js(self):
     # Returns the JS files required by this page and its StreamField blocks.
     @property
     def media_js(self):
-        return sorted(set(self.page_js + self.streamfield_media("js")))
+        return list(dict.fromkeys(self.page_js + self.streamfield_media("js")))
```

Comment:
This change allows us to preserve the order of JS script loads for a block, for example given a block like

```py
class MyBlockWithJS(blocks.StructBlock):
    class Media:
        js = ["b.js", "a.js"]
```

Our template will currently sort these and generate tags like

```html
<script src="/static/js/routes/on-demand/a.js"></script>
<script src="/static/js/routes/on-demand/b.js"></script>
```

This causes problems if something in a.js relies on something in b.js being loaded first, [as is the case with wagtailcharts](https://github.com/overcastsoftware/wagtailcharts/blob/f45f9379166a229158119f7bc9bf553c0c1f3c1b/wagtailcharts/templates/wagtailcharts/tags/render_charts.html#L4-L7).

I suppose in theory this change might break some alphabetized dependency we didn't know we needed, but I think it's better to be explicit with ordering.

Reply:
The only place I thought this might cause an issue is with our [video player](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/v1/atomic_elements/organisms.py#L773) loading youtube's iframe API but the loading order doesn't matter because all the [youtube script](https://www.youtube.com/iframe_api) does is load a second script that explicitly has an `async` attribute.

So this change looks a-okay. 👍